modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -5,70 +5,71 @@ endif::[]
 :keywords: cloudhub, managing, monitoring, troubleshooting, runtime manager, arm, applications, servers
 
 image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
-image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
+image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid"]
 image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
-image:logo-pcf-active.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
+image:logo-rtf-disabled.png[link="/runtime-fabric/deploy-to-runtime-fabric", title="Runtime Fabric"]
 
 
-If you ever experience problems with your deployed apps, the Runtime Manager offers several tools that can help you get to the root cause of it.
 
-[NOTE]
-Depending on the xref:deployment-strategies.adoc[Deployment Strategy] you're using, there is a different set of possible troubleshooting resources available. This feature gap will be gradually reduced as new releases implement the missing features on either side.
+If you experience problems with your deployed apps, Runtime Manager provides several tools that can help you determine the root cause.
+
+Depending on your deployment strategy, there is a different set of possible troubleshooting resources available. 
 
 See xref:monitoring.adoc[Monitoring] for details on the monitoring tools of the Runtime Manager, which can also be of help you diagnose an issue.
```

Comment:
Do we really want to follow a statement about troubleshooting with what amounts to an aside about monitoring?